ga4gh/datamodel/ontologies.py

```diff
@@ -8,36 +8,45 @@
 import ga4gh.protocol as protocol
 
 
-class Ontology(object):
+class OntologyTermMap(object):
     """
-    At this moment an "Ontology" is just a map between names and IDs (e.g.
-    in Sequence Ontology we would have "SO:0001583 <-> missense_variant")
-    This is a tempotrary solution and must be replaced by more comprehensive
-    ontology object.
+    A bidectional map between ontology names and IDs (e.g. in Sequence
+    Ontology we would have "SO:0001583 <-> missense_variant"). This
+    implementation uses a tab separated TXT file: "id\tname"
 
-    This implementation uses a tab separated TXT file: "id\tname"
+    This is a temporary solution and must be replaced by more comprehensive
+    ontology object.
     """
     def __init__(self, localId):
         self._localId = localId
         self._dataUrl = None
+        self._sourceName = None
         self._nameIdMap = dict()
         self._idNameMap = dict()
 
     def populateFromFile(self, dataUrl):
         """
         Populates this ontology map from the specified dataUrl.
+        This reads the ontology term name and ID pairs from the
+        specified file.
         """
         self._dataUrl = dataUrl
-        self.readOntology(self._dataUrl)
+        # TODO shouldn't sourceName be the name of the ontology, (e.g,
+        # sequence_ontology?)
```

Comment:
Yes, I think so. This does need to be specified better, using the user provided name makes enough sense.

To do proper relational management we should probably give unique identifiers to the ontology maps. Perhaps we should provide a mechanism to specify the ontology map for a given variant set?